Core/MemMap.cpp

```diff
@@ -533,19 +533,16 @@ bool HandleFault(uintptr_t hostAddress, void *ctx) {
 			// It was a read. Fill the destination register with 0.
 			// TODO
 		}
-		// Move on to the next instruction.
+		// Move on to the next instruction. Note that handling bad accesses like this is pretty slow.
```

Comment:
We could set a flag to rebuild jit with slow memory, either for this block or all blocks.  Might be faster overall.

-[Unknown]